src/shared_modules/utils/stringHelper.h

```diff
@@ -310,7 +305,43 @@ namespace Utils
 
         return !str.empty() && it == str.end();
     }
-}
+
+    static int parseStrToBool(const std::string& str)
+    {
+        return (str.compare("yes") == 0) ? 1 : (str.compare("no") == 0) ? 0 : -1;
+    }
+
+    static long parseStrToTime(const std::string str)
+    {
+        std::size_t pos;
+        try
+        {
+            auto seconds {std::stoll(str, &pos)};
```

Comment:
If "parseStrToTime" returns a "long" type, then "stol" function should be used instead of "stoll"

Reply:
Fix it =)